README.md

```diff
@@ -1,74 +1,8 @@
-# Capital Framework [![Travis](https://img.shields.io/travis/cfpb/capital-framework.svg?style=flat-square)](https://travis-ci.org/cfpb/capital-framework) [![npm](https://img.shields.io/npm/v/capital-framework.svg?style=flat-square)](https://www.npmjs.com/package/capital-framework)
+# Capital Framework
 
-:warning: **Heads up:** In an effort to unify our design resources, we will disable this repo and migrate Capital Framework's code to [cfpb/design-system](https://github.com/cfpb/design-system) in Summer 2019. Until the migration occurs, please continue to use this repo.
+# :warning: THIS REPO IS DEPRECATED :warning:
```

Comment:
Maybe add a date stamp. I guess that's dictated by the commit stamp, but like https://github.com/cfpb/cf-buttons#warning-this-repo-is-deprecated-1152016-warning can be nice to see at a glance.